src/store/githubOauth.ts

```diff
@@ -0,0 +1,9 @@
+let oauth: string;
```

Comment:
```suggestion
let oauth: string | undefined;
```

Reply:
`| undefined` is not necessary.

If `.env` does not have a `ghOauth` field, will error due to not matching `dotEnv` specification.

If `.env` does have `ghOauth` and not defined: `ghOauth=`, bot will still work under normal API limits, with no errors.

Reply:
There's a few issues that can arise, example [here](https://www.typescriptlang.org/play?#code/DYUwLgBAHgXBB2BXAtgIxAJwNwCgcDNF4BjMASwHt4Iz4ywANACgEoIBvHCb6CAXggBWXAF88tesxa4cZfBCZQWXHsSoBnCqAB0wCgHNF0oA)

Reply:
Implemented as per discussion on discord.